styles.css

```diff
@@ -0,0 +1,108 @@
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  max-width: 350px;
```

Comment:
Kindly, you should not limit the size of your site to a specific size, it should always display full screen.